plugins/ssl/ssl-certificate-expiry

```diff
@@ -123,10 +126,33 @@ print_expire_days() {
     [ -n "$starttls" ] && s_client_args="-starttls $starttls"
 
     # shellcheck disable=SC2086
```

Comment:
I guess, you need to move this line below your comment - otherwise `shellcheck -s dash PLUGIN_FILENAME` will complain. It may look like a comment, but it is a line interpreted by shellcheck.